.generator/templates/model/ObjectSerializer.mustache

```diff
@@ -197,14 +199,15 @@ export class ObjectSerializer {
                     try {
                         oneOfs.push(ObjectSerializer.deserialize(data, oneOf, format))
                     } catch (e) {
-                        console.debug(`could not deserialize ${oneOf} (${e})`)
+                        if (e instanceof ParseException) {
+                            if (e.type == "enum") {
```

Comment:
I'm a bit confused by this check. Why not use `unparsedObject` instead? Not sure I completely follow the logic.

Reply:
We allow pushing oneOfs that fails because of an invalid enum value, after that if 0 or multiple oneOfs match, then we raise an exception to be caught in the parent object call "Object.deserialize(oneOfField)" and put in the parent's unparsedObject.

Reply:
How is that any different than before. You're basically just removing a log line, this check does nothing, since if the exception is not ParseException, nothing is being done either, which is the same as `continue` since it's the end of the loop anyway.

Reply:
Yes, when I added this I had other plans in mind, this should be removed.